demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -248,6 +248,7 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
           }
           await this.openAudioOutputFromSelection();
           this.hideProgress('progress-authenticate');
+          await this.createLogStream();
```

Comment:
Should this go to initializeMeetingSession when we intialize MeetingPostSessionLogger. There is no reason to create log stream for local host.
```
    if (location.hostname === 'localhost' || location.hostname === '127.0.0.1') {
      logger = consoleLogger;
    } else {
      logger = new MultiLogger(
        consoleLogger,
        new MeetingSessionPOSTLogger(
          'SDK',
          configuration,
          DemoMeetingApp.LOGGER_BATCH_SIZE,
          DemoMeetingApp.LOGGER_INTERVAL_MS,
          `${DemoMeetingApp.BASE_URL}logs`,
          logLevel
        ),
      );
    }
```